src/wrapper/simulation_object_updating_component.py

```diff
@@ -73,4 +90,41 @@ def next_tick(self, tick: int):
             simulation_object.update(subscription_results[simulation_object.traci_id])
 
     def _fetch_initial_simulation_objects(self):
-        pass
+        folder = dirname(self._sumo_configuration)
+        inputs = next(sumolib.xml.parse(self._sumo_configuration, "input"))
+        net_file = join(folder, inputs["net-file"][0].getAttribute("value"))
+        additional_file = join(
+            folder, inputs["additional-files"][0].getAttribute("value")
+        )
+        net = sumolib.net.readNet(net_file)
+        platforms = list(sumolib.xml.parse(additional_file, "busStop")) + list(
+            sumolib.xml.parse(additional_file, "trainStop")
+        )
+
+        # Tracks
+        self._simulation_objects += [
+            Track.from_simulation(edge, self) for edge in net.getEdges()
+        ]
+
+        # self._simulation_objects += [
+        #    Node.from_simulation(node, self)
+        #    for node in net.getNodes()
+        #    if len(node.getConnections()) < 3
+        # ]
```

Comment:
Will you need this comment later? 🙃

Reply:
I am currently unsure if we need the nodes anyway (e.g. if we don't have a signal at a junction between two edges).

Reply:
You can resolve this conversation if it's not needed 😄